UXO_Kismet_SF_3.tex

```diff
@@ -0,0 +1,363 @@
+\input{common/standard_preamble}
+
+\title{
+\Huge Kismet Hybrid Rocket Engine\\
+Static Fire 3\\
+\vspace{1cm}
+\Large Static Fire Test Operations Procedures}
+
+%begin operators definitions -- {{{
+\operator{ops}{Blue}{OPS}{Operations Director}
+\operator{primary}{Fuchsia}{PRIMARY}{Primary Fill Operator}
+\operator{secondary}{Emerald}{SECONDARY}{Secondary Fill Operator}
+\operator{daq}{OliveGreen}{DAQ}{DAQ Technician}
+\operator{control}{Maroon}{CONTROL}{Control System Operator}
+\operator{heat}{orange}{HEAT}{Heating Technician}
+\operator{peri}{red}{P1}{Perimeter Guard 1}
+\operator{perii}{red}{P2}{Perimeter Guard 2}
+\operator{periii}{red}{P3}{Perimeter Guard 3}
+%end operators definitions -- }}}}}
```

Comment:
Has the decision been made as to who each of these people are?

Reply:
Perimeter specifically, or everyone?

Reply:
everyone really. Might be cool (unnecessary, but kinda historically cool) to leave a comment with who's being what role.